### PR TITLE
Added link to the discord server inside FAQ.

### PR DESCRIPTION
**Pull request by raffaellasuardini**
# Trac ticket number
N/A

# Branch description
Added a link for Discord Django server on the FAQ page.

# Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [x] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

**Comment by raffaellasuardini** [comment]
@bmispelon

**Comment by bmispelon** [comment]
✅  I am Baptiste and I approve of this pull request 🚀

File: docs/faq/help.txt [hunk 1]
```diff
@@ -22,11 +22,13 @@ Then, please post it in one of the following channels:
 * The Django Forum section `"Using Django"`_. This is for web-based
   discussions.
 * The |django-users| mailing list. This is for email-based discussions.
+* The `Django Discord server`_ for chat-based discussions.
```
Comment (sarahboyce) [inline docs/faq/help.txt:25]: I noticed that the Django Forum recommended the "Using Django" section. Do we want to recommend a channel for getting help? I know the Discord does "welcome" you a bit so might not be needed 🤔 
What do you think? (cc @knyghty)

**Review by sarahboyce — COMMENTED** [review]
Thank you for this @raffaellasuardini ⭐

File: docs/faq/help.txt (hunk 1 above)
Comment (knyghty) [reply docs/faq/help.txt:25]: There is a #getting-started channel but it's probably not the right channel for _everyone_ coming through here. The channels are also a bit more subject to change than the forum, so probably best to not point to anything specific but let the server onboarding flow do the work.

**Review by sarahboyce — APPROVED** [review]
Thank you @raffaellasuardini ⭐ this is great
I pushed a very minor update to mention the Discord in the list of "please don't publicly disclose security issues" section

Welcome onboard ⛵

**Pull request merged**